amenity-points.mss

```diff
@@ -765,21 +769,34 @@
     marker-clip: false;
   }
 
-  [feature = 'historic_memorial'][zoom >= 17] {
+  [feature = 'historic_memorial'][zoom >= 17],
+  [feature = 'historic_memorial_stele'][zoom >= 19] {
```

Comment:
Do we have `historic_memorial_stele` defined in the `project.mml`? I guess no and we use simple memorial check now.

Reply:
I also think this should be rather z18+.

Reply:
both fixed